slynk/slynk.asd

```diff
@@ -19,17 +19,22 @@
 
 (defsystem :slynk
   :serial t
+  ;; If/when we require ASDF3, we can use UIOP:DEFINE-PACKAGE to avoid package
```

Comment:
Instead of this ASDF3 note, I would like to see here explained why SBCL needs this muffling in the first place.

If I understand correctly it's because ASDF, by default, will refuse to write `.fasl` for compilation units that signalled `WARNING` during `COMPILE-FILE`? Why does only SBCL need this?